lms/envs/aws.py

```diff
@@ -209,6 +209,8 @@
 CONTACT_EMAIL = ENV_TOKENS.get('CONTACT_EMAIL', CONTACT_EMAIL)
 BUGS_EMAIL = ENV_TOKENS.get('BUGS_EMAIL', BUGS_EMAIL)
 PAYMENT_SUPPORT_EMAIL = ENV_TOKENS.get('PAYMENT_SUPPORT_EMAIL', PAYMENT_SUPPORT_EMAIL)
+PAYMENT_SUPPORT_PHONE = ENV_TOKENS.get('PAYMENT_SUPPORT_PHONE', PAYMENT_SUPPORT_PHONE)
```

Comment:
Stanford-specific settings go in our subdirectory: `openedx/stanford/lms/envs/aws.py`